.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: CI (Python, Docker Build)
+
+on: [ push, pull_request ]
+
+jobs:
+  build-image:
+    runs-on: ubuntu-latest
+    if: github.ref != 'refs/heads/main'
+    steps:
+      - name: Echo tag
+        id: echotag
+        run: |
+          echo "Building an image with the following tag:"
```

Comment:
I guess you could say it once since it's the same thing?
`echo "Building and image with the following tag based on the same commit` or something

Reply:
yeah fair point, fixed